voteswap/templates/base.html

```diff
@@ -32,8 +32,10 @@
         </div>
       </div>
     </nav>
-    {% block main %}
-    {% endblock %}
+    <div class="container">
```

Comment:
what page doesn't need it? The homepage falls apart without it (as would all pages im aware of).